examples/kubernetes.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: red
+  namespace: default
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: red
+    spec:
+      containers:
+      - name: red
+        image: segment/red:v0.2.0
+        args: [
+            "proxy",
+            "-upstream", "consul://consul.default.svc.cluster.local:8500/redis?cluster=dev",
```

Comment:
Yep, right now on dev k8s  Consul is setup as a StatefulSet + Service. We will have to add a DaemonSet to deploy the agent on every box.